main.py

```diff
@@ -168,7 +168,7 @@ def get_market():
 
 def get_download_path():
     # By default images are saved to '/home/[user]/[Pictures]/BingWallpapers/'
-    default_path = check_output("xdg-user-dir PICTURES", shell=True).strip() + "/BingWallpapers"
+    default_path = check_output("xdg-user-dir PICTURES", shell=True).strip().decode("utf-8") + "/BingWallpapers"
```

Comment:
Maybe I should add a comment for this in the file? The .strip() is necessary in Python 2.7, because check_output otherwise includes a line break. The .decode("utf-8") is necessary in Python 3 because check_output for some stupid reason has the type "bytes".